RFC-001-BAM.md

```diff
@@ -247,13 +243,14 @@ A header's key acts as its identifier. A header's value MUST encode the followin
 
 2. A value: The actual value that is associated with the header
 
-    A header's value MUST be a single token, e.g. no nested structure etc. This is to facilitate comparisons of header-values. If a value needs to parameterized further, `parameters` should be used. Every header MUST have a value.
+    Every header MUST have a value. A header's value determines the parameters.
 
 3. Parameters
 
-    Parameters are to be used to further specify a header's value. Parameters are a key-value structure with **unique** keys. Similar to the actual header value, both keys and values must be single tokens. Parameters are optional and may therefore be empty or left out if the encoding allows this.
-
-Headers can occur multiple times, e.g. a single header-key can map to multiple values where all of them have one `value` and `parameters`. How this is represented depends on the encoding.
+    Parameters are additional data that depend on a header's value.
+    They might be optional or also empty if the header's value doesn't define any parameters.
+    If the encoding supports this, parameters might even be completely left out.
+    In this case, implementations should treat the header as if the empty set of parameters was sent.
```

Comment:
🔧
"A parameter for a given header value may be optional or mandatory. Optional parameters may be omitted. If the header value specifies no mandatory parameters, then the parameters section may be completely left out and implementations should treat this as if the empty set of parameters was sent"